src/wazuh_db/helpers/wdb_agents_helpers.c

```diff
@@ -137,18 +137,29 @@ cJSON* wdb_insert_vuln_cves(int id,
     cJSON_AddStringToObject(data_in, "updated", updated);
     cJSON_AddBoolToObject(data_in, "check_pkg_existence", check_pkg_existence);
 
+    // Limiting references just in case there are too many or the links are too long
+    char* str_cvs_references = NULL;
+    os_calloc(WDB_MAX_QUERY_SIZE, sizeof(char), str_cvs_references);
     cJSON *j_cvs_references = cJSON_CreateArray();
     int refcount;
     for (refcount = 0; external_references[refcount]; ++refcount)
     {
         cJSON *j_ref_item = cJSON_CreateString(external_references[refcount]);
         cJSON_AddItemToArray(j_cvs_references, j_ref_item);
+
+        cJSON_PrintPreallocated(j_cvs_references, str_cvs_references, WDB_MAX_QUERY_SIZE, FALSE);
```

Comment:
Could we include this logic in a loop while the refs length is bigger than `VULN_CVES_MAX_REFERENCES`?

Something like:
```
while (strlen(str_cvs_references) >= VULN_CVES_MAX_REFERENCES) {
    // Remove an entry from the array
}
```